### ISSUE TITLE
Zeus not working with Achilles

### ISSUE THREAD

**Issue by FeldOtto55** [oOKexOo/AresModAchillesExpansion#103]
**Hi!
My problem is when I start a game as zeus .. it's not working .. I just turn into a ghost kinda and I can only run around like a midget :D Can't do anything, no interface at all except for the stance indicator up top.**

![Issue while being Zeus](https://user-images.githubusercontent.com/30060468/28044487-f10e825c-65d7-11e7-90dc-ab83470d6c91.JPG)

**I tried re-installing my game and only installing CBA (v3.4.0) and Ares Mod - Achilles Expansion (0.0.8)
but no luck :/**

![Used Expansions/Mods](https://user-images.githubusercontent.com/30060468/28044485-eed5435e-65d7-11e7-990a-ba62525fb7e2.JPG)

_It's working perfectly fine with only Ares (not achilles) but it lacks certain options that I would like to use in Achilles._

**Did anyone else had this kind of truble? Do you know some kind of a solution? I've been looking around the internet for about a week or two now and I could not figure it out so I thought I give it a try and ask you guys.**

**Comment by FeldOtto55** [comment]
![Used Expansions/Mods](https://user-images.githubusercontent.com/30060468/28044485-eed5435e-65d7-11e7-990a-ba62525fb7e2.JPG)
![Issue while being Zeus](https://user-images.githubusercontent.com/30060468/28044487-f10e825c-65d7-11e7-90dc-ab83470d6c91.JPG)

**Comment by CreepPork** [comment]
I'm more putting my money on that your mission file is misconfigured. It looks like you are a Game Logic that has no Zeus interface opened, if you can attach the whole mission and I'll have a look.

**Comment by FeldOtto55** [comment]
I simply put down a **Zeus Game Master modul** and a **playable soldier** .. make the GameMaster's owner **#adminLogged** .. Last time I made it so that the Zeus Game Master forces the Zeus interface ..  this time I didn't. It spawns you down as a soldier .. but even the spawning is fucked up .. you can't select spawn location and can'trespawn at all with Achilles mod in but it works with Ares and Vanila without any problem .. checked both of them with this exact same mission :/
Here is the mission https://www.dropbox.com/s/ccmje64h231gl9f/AchillesZeus.Stratis.7z?dl=0

**Comment by FeldOtto55** [comment]
If you could help me that would be amazing cus i'm out of ideas :'(

**Comment by CreepPork** [comment]
I remember looking into some issues with #adminLogged not working. Try just putting in the variable name for the curator's game logic. That should make it work.

If it does work then #adminLogged is broken.

**Comment by FeldOtto55** [comment]
UPDATE .. It works in .. singleplayer .. but not in multiplayer .. :/

**Comment by oOKexOo** [comment]
I just tried your mission on a dedicated server. CBA+Achilles 0.0.8c on client side and no mods on server side. I could access the Zeus interface without any issues after logging in.
You could try loading Achilles to the server. The initialization is typically more stable by doing so.

Btw. the other modules (e.g. restricted editing area) you placed won't work, since you did not specify the curator module for them.

**Comment by FeldOtto55** [comment]
Hmmm ... why is it different than Vanilla? I played in Vanilla and Ares for weeks without problem and the **Cost Module** and the **Restriction around player Module** worked just fine .. I did everything as any other day .. and if I disable Achilles, everything works 100% just fine .. so what is different with Achilles?

**Comment by oOKexOo** [comment]
The module initialization in Ares is different from Achilles, that's why.
Ares adds the modules manually to the interface, whereas Achilles modules are added by the hard-coded Zeus initialization. The latter is more stable, however it does not work if modules from thrid-party add-ons are not allowed. Hence, there is a workaround that forces the game to accept Achilles nonetheless. Unfortunatly, this workaround requires reloading the Zeus interface, which is not very stable and seems to fail in your case.
Hence, make sure the workaround is not required by loading Achilles to the server.

**Comment by FeldOtto55** [comment]
I see .. and how could I do that?

**Comment by oOKexOo** [comment]
Wait a minute. It's not a dedicated server... What are you exactly doing? Do you test it in SP or as a server host?

**Comment by oOKexOo** [comment]
So you are not testing it on a dedicated server?
I actually still have no idea why it is a persistent problem for you. It's not reproducible for me.
I might still miss something. Can you summarize what you did in a proper form (check bottom of descriptions in [Steam](http://steamcommunity.com/sharedfiles/filedetails/?id=723217262)).

**Comment by FeldOtto55** [comment]
:'( I'm too un-informed to know these :'( 
I have Arma 1.66 (have to use this old cracked version cus my friends don't have the game and when we play from time to time in Lan .. we play with this or CoD4 :D ) ((dunno if that would cause the problem tho cus a friend of mine have Arma 3 on steam .. and I asked him to do everything I did .. and it dosen't work for him either))
Mods I downloaded [CBA v3.4](http://www.armaholic.com/page.php?id=18767) , [Ares Mod Achilles Expansion 0.0.8](https://forums.bistudio.com/forums/topic/191113-ares-mod-achilles-expansion/)
I put these into the Documents/Arma 3/  folder
Activate it in the Launcher .. and start the game .. go into multiplayer and Host Server .. and that is all
Any progress on knowing what could be the problem ? :/

**Comment by oOKexOo** [comment]
Well, that explains a lot... You know that Achilles is constantly updated and makes use of feature introduced in newer versions of Arma.
Moreover, I won't help you with thtat cracked version. At least you were honest, so I kindly remind you that there are older versions of Achilles on this website.

**Comment by FeldOtto55** [comment]
Okey .. I'll try with older versions and we'll see .. 
Will try to make my friends to buy Arma 3 too cus it's a LOT of fun :)
Thanks for your help and patience
